examples/random-graph-distributed/random-graph.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Sprotty Mindmap Example</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="css/page.css">
+    <!-- support Microsoft browsers -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/dom4/3.0.0/dom4.js">
+</head>
+<body>
+    <div class="container">
+        <div class="row" id="sprotty-app" data-app="random-graph-distributed">
+            <div class="col-md-10">
+                <h1>Sprotty Random Graph Example</h1>
+            </div>
+            <div class="help col-md-2">
+                <a href='https://github.com/eclipse/sprotty/wiki/Using-sprotty'>Help</a>
+            </div>
+        </div>
+        <div class="row">
```

Comment:
It may help to add a paragraph here about how to download and start the server, something like:

If no diagram shows up here, you probably haven't started the elk-server yet.
Please follow the steps below:
1. Download [elk-server](https://github.com/TypeFox/elk-server/releases/tag/v0.1.0)
2. Run the application with the `--socket` argument:
   ```
    elk-server-0.1.0/bin/elk-server --socket
   ```
3. Refresh this page